effcee/match.cc

```diff
@@ -215,7 +215,7 @@ Result Match(StringPiece input, StringPiece checks, const Options& options) {
           for (auto j = first_unresolved_negative,
                     limit = std::min(first_unresolved_dag, i);
                j < limit; ++j) {
-            resolved[i] = true;
+            resolved[j] = true;
```

Comment:
OMG.  This is the obvious correct fix.
... and is an argument for using std:: transformers rather than manual iteration over a loop. So embarrasing.